docs/kyma/docs/04-02-local-installation.md

```diff
@@ -20,140 +16,132 @@ Additionally, download these tools:
+  ```
 
-This script sets up default parameters, starts Minikube, builds the Kyma Installer, generates local configuration, creates the Installation custom resource, and sets up the Installer.
+6. Kyma comes with a local wildcard self-signed `server.crt` certificate. Add this certificate to your OS trusted certificates to access the Console UI. On MacOS, run:
+  ```
+  sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain certs/workspace/raw/server.crt
+  ```
+  >**NOTE:** Mozilla Firefox uses its own certificate keychain. If you want to access the Console UI though Firefox, add the Kyma wildcard certificate to the certificate keychain of the browser. To access the Application Connector and connect an external solution to the local deployment of Kyma, you must add the certificate to the trusted certificate storage of your programming environment. Read [this](/components/application-connector#details-access-the-application-connector-on-a-local-kyma-deployment) document to learn more.
+
+7. Start the installation. Trigger the `run.sh` script to start Minikube with a Kyma-specific configuration and install the necessary components. Define the password used to log in to the Console UI using the `--password` flag. Run:
+  <div tabs>
+    <details>
+    <summary>
+    MacOS
+    </summary>
+
+    ```
+    ./cmd/run.sh --password {USER_PASSWORD}
+    ```
+    </details>
+    <details>
+    <summary>
+    Linux
+    </summary>
+
+    ```
+    ./cmd/run.sh --password {USER_PASSWORD} --vm-driver virtualbox
+    ```
+    </details>
+  </div tabs>
+
+8. By default, the Kyma installation is a background process, which allows you to perform other tasks in the terminal window. Nevertheless, you can track the progress of the installation by running the `is-installed.sh` script. It is designed to give you clear information about the Kyma installation. Run it at any point to get the current installation status, or to find out whether the installation is successful.
 
-> **NOTE:** See [this](#installation-local-installation-scripts-deep-dive) document for a detailed explanation of the `run.sh` script and the subscripts it triggers.
+  ```
+  ./scripts/is-installed.sh
+  ```
+  >**TIP:** If the script indicates that the installation failed, try to install Kyma again by re-running the `run.sh` script. If the installation fails in a reproducible manner, don't hesitate to create a [GitHub](https://github.com/kyma-project/kyma/issues) issue in the project or reach out to the ["installation" Slack channel](https://kyma-community.slack.com/messages/CD2HJ0E78) to get direct support from the community.
 
-You can execute the `installation/cmd/run.sh` script with the following parameters:
+9. After the installation is completed, you can access the Console UI. Go to [this](https://console.kyma.local) address and select **Login with Email**. Use the **admin@kyma.cx** email address and the password you set using the `--password` flag.
```

Comment:
We can point to the troubleshooting section in case the user did not set the password using the flag. What do you think?

Reply:
The document is riddled with notes and tips, I'd rather not add another one to avoid making it cluttered. I'll make the password troubleshooting tip more visible by moving it to the first position in the "Troubleshooting" section.